packages/flutter/lib/src/material/switch_list_tile.dart

```diff
@@ -409,6 +411,15 @@ class SwitchListTile extends StatelessWidget {
   /// If this property is null then its value is based on [ListTileTheme.dense].
   final bool dense;
 
+  /// The tile's internal padding.
+  ///
+  /// Insets a [SwitchListTile]'s contents: its [title], [subtitle],
+  /// [secondary], and [Switch] widgets.
+  ///
+  /// If null, [ListTile]'s default of `EdgeInsets.symmetric(horizontal: 16.0)`
```

Comment:
I think we default to ListTileTheme here, and that defaults to ...16.0

Reply:
It looks like the `ListTile` actually defines the default here:

https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/list_tile.dart#L860